x-pack/plugins/infra/server/lib/alerting/common/utils.ts

```diff
@@ -75,3 +77,8 @@ export const createScopedLogger = (
     },
   };
 };
+
+export const getViewInAppUrl = (basePath: IBasePath) =>
+  basePath.publicBaseUrl
+    ? new URL(basePath.prepend(LINK_TO_METRICS_EXPLORER), basePath.publicBaseUrl).toString()
+    : LINK_TO_METRICS_EXPLORER;
```

Comment:
For the logs app link in #127890 the equivalent function is located in https://github.com/fkanout/kibana/blob/680d6c11c69ca5b85d016a2ee33812888941adc1/x-pack/plugins/infra/common/formatters/alert_link.ts#L9 and called `getLogsAppAlertUrl`. Don't we want to apply the same pattern here in terms of location, name and arguments?

Reply:
Fixed